code/game/objects/items/crayons.dm

```diff
@@ -775,7 +775,7 @@
 /obj/item/toy/crayon/spraycan/add_context(atom/source, list/context, obj/item/held_item, mob/living/user)
 	. = ..()
 
-	if(!user.can_perform_action(src, NEED_DEXTERITY|NEED_HANDS))
+	if(!user.can_perform_action(src, NEED_DEXTERITY|NEED_HANDS|SILENT_ADJACENCY))
```

Comment:
Snuck in from your other PR? Gotta clean your master branch.

Reply:
WHOOOPS.